jps-plugin/src/org/intellij/erlang/jps/builder/ErlangModuleBuildOrder.java

```diff
@@ -17,22 +17,13 @@
 package org.intellij.erlang.jps.builder;
 
 import com.intellij.util.containers.ContainerUtil;
-import com.intellij.util.xmlb.annotations.AbstractCollection;
-import com.intellij.util.xmlb.annotations.Attribute;
 import com.intellij.util.xmlb.annotations.Tag;
 
 import java.util.List;
 
 @Tag("module")
-public class ErlangModuleBuildOrderDescriptor {
-  @Attribute("name")
-  public String myModuleName = "";
+public class ErlangModuleBuildOrder {
+  public List<String> myOrderedErlangFilePaths = ContainerUtil.newArrayList();
 
-  @Tag("erlangModules")
-  @AbstractCollection(surroundWithTag = false)
-  public List<ErlangFileDescriptor> myOrderedErlangFilePaths = ContainerUtil.newArrayList();
-
-  @Tag("testErlangModules")
-  @AbstractCollection(surroundWithTag = false)
-  public List<ErlangFileDescriptor> myOrderedErlangTestFilePaths = ContainerUtil.newArrayList();
+  public List<String> myOrderedErlangTestFilePaths = ContainerUtil.newArrayList();
```

Comment:
I'm talking about ```@Tag("module")``` annotation - it's our XML serializer's annotation. We don't need it there.